lib/floe/workflow/path.rb

```diff
@@ -34,7 +34,18 @@ def value(context, input = {})
         return obj if path == "$"
 
         results = JsonPath.on(obj, path)
-        results.count < 2 ? results.first : results
+        case results.count
+        when 0
+          raise Floe::PathError, "Path [#{payload}] references an invalid value"
```

Comment:
This one change has big consequences.

Before: A path pointing to nothing would just treat the value as `nil`
After: A path pointing to nothing is a runtime error